src/components/Menu.vue

```diff
@@ -83,7 +119,7 @@
           Tutorial
         </ion-list-header>
         <ion-menu-toggle :auto-hide="false">
-          <ion-item button @click="openTutorial()">
+          <ion-item button :detail="false" @click="openTutorial()">
```

Comment:
When clicking on the Tutorial link, I don't get navigate to the full screen Tutorial page. The page still shows the sidebar.

Actual:
![Screenshot 2025-04-17 at 11 24 30 AM](https://github.com/user-attachments/assets/361392f5-c25c-45c4-8391-1672f1ed4a86)

Expected:
![Screenshot 2025-04-17 at 11 24 47 AM](https://github.com/user-attachments/assets/35dafe6c-8074-4273-934f-2b58d573138e)

Reply:
Should be fixed now: https://github.com/ionic-team/ionic-vue-conference-app/pull/55/commits/4ccaab9328c48d8ea59f79cdbd7b67025b16b412